bundles/org.openhab.binding.innogysmarthome/src/test/java/org/openhab/binding/innogysmarthome/internal/handler/InnogyBridgeHandlerTest.java

```diff
@@ -50,6 +55,12 @@ public class InnogyBridgeHandlerTest {
 
     @BeforeEach
     public void before() throws Exception {
+        final Logger loggerBridge = (Logger) LoggerFactory.getLogger(InnogyBridgeHandler.class);
+        loggerBridge.setLevel(Level.OFF);
+
+        final Logger logerBaseHandler = (Logger) LoggerFactory.getLogger(BaseThingHandler.class);
+        logerBaseHandler.setLevel(Level.OFF);
```

Comment:
I have a doubt but after checking few other bindings are doing the same in their tests.
I am not sure whether this is a good practice or not ?
The idea is to hide some logs because they are not important for these tests ?

Reply:
Yes, tests should not have any output except for assertions as the tests are automated. It would greatly reduced the output of the build process and make it somewhat simpler to focus on the relevant output.
If an assertion fails, the binding developer should fix/debug the test and can turn it on or set breaks or whatever.

Reply:
Ok, makes sense.